zerver/lib/push_notifications.py

```diff
@@ -253,6 +263,11 @@ def parse_gcm_options(options: Dict[str, Any], data: Dict[str, Any]) -> str:
 
     return priority  # when this grows a second option, can make it a tuple
 
+def send_android_encrypted_push_notification(encrypted_data: EncryptedData,
+                                             options: Dict[str, Any], remote: bool=False) -> None:
+    for device, payload in encrypted_data:
+        send_android_push_notification([device], payload, options, remote)
+
```

Comment:
As a matter of style and code clarity, if we do have this loop, the code would be easier to read if it's simply inlined at the one place this function is used -- this function doesn't really add anything. (But as mentioned in a separate comment, we should arrange the internal API so that we don't need this loop at all.)

Reply:
agreed